sympy/assumptions/ask.py

```diff
@@ -1233,7 +1233,6 @@ def ask(proposition, assumptions=True, context=global_assumptions):
     >>> ask(Q.even(x*y), Q.even(x) & Q.integer(y))
     True
     >>> ask(Q.prime(x*y), Q.integer(x) &  Q.integer(y))
-    False
```

Comment:
This is not a good example for the docstring. It actually would be good to have an example that returns `None` here, but this one is a bit of a "gotcha" that people might not understand at first glance (especially since we even got it wrong). How about something much simpler, like `ask(Q.positive(x - y), Q.positive(x) & Q.positive(y))` (or feel free to suggest another one).